README.md

```diff
@@ -33,9 +33,9 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📊 Morse Code <a name="about-project"></a>
+# 📊 enumerables-rb <a name="about-project"></a>
 
-A Morse Code Decoder with Ruby
+A enumerables-rb Decoder with Ruby

```

Comment:
- Please note that in the getting started section of your README the project information does not match the current project. Kindly fix this 😉